src/compose/ComposeBox.js

```diff
@@ -233,7 +233,7 @@ class ComposeBox extends PureComponent<Props, State> {
     const subject = topic !== editMessage.topic ? topic : undefined;
     if (content || subject) {
       updateMessage(auth, { content, subject }, editMessage.id).catch(error => {
-        showErrorAlert(error.message, 'Failed to edit message');
```

Comment:
I wonder if we should be setting `error.message` correctly instread. Also, is `error` ever not defined?

Reply:
> error ever not defined?

Just wanted to be extra safe 😅

Reply:
> I wonder if we should be setting error.message correctly instread.

@gnprice Thoughts?

Reply:
> > error ever not defined?
> 
> Just wanted to be extra safe 😅

Don't be :)
In fact, any check, in this case, might be 'too safe' as the only exception we are throwing is an API exception.

Reply:
> Don't be :)

Okay, cool! updated 👍